ticket.go

```diff
@@ -4,8 +4,10 @@ import (
     "strconv"
 )
 
-// Ticket Struct
-type Ticket struct {    
+// Ticket Structure
```

Comment:
Comment is redundant, `Ticket struct` is pretty clear

Reply:
Fair enough, removed